BraintreeDropIn/BTCardFormViewController.m

```diff
@@ -274,6 +286,55 @@ - (void)updateRequiredFields {
     }
 }
 
+#pragma mark - card.io
+
+- (void)setupCardIO {
+    if ([self isCardIOAvailable]) {
+        BTUIKInputAccessoryToolbar *cardToolBar = [BTUIKInputAccessoryToolbar new];
+        UIBarButtonItem *flexSpace = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemFlexibleSpace target:nil action:nil];        
+        UIBarButtonItem *cardIOButton = [[BTUIKBarButtonItem alloc] initWithTitle:BTUIKLocalizedString(SCAN_CARD_IO_ACTION) style:UIBarButtonItemStylePlain target:self action:@selector(presentCardIO)];
+        cardToolBar.items = @[flexSpace, cardIOButton, flexSpace];
+        self.cardNumberField.textField.inputAccessoryView = cardToolBar;
+    }
+}
+
+- (BOOL)isCardIOAvailable {
```

Comment:
Agreed. I decided to not make it opt-in. We can add an option to opt-out when it comes up.